bears/natural_language/AlexBear.py

```diff
@@ -18,6 +21,30 @@ class AlexBear:
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
 
+    @classmethod
+    def check_prerequisites(cls):  # pragma: no cover
+        parent_prereqs = super().check_prerequisites()
+        if parent_prereqs is not True:
+            return parent_prereqs
+
+        incorrect_pkg_msg = (
+            "Please ensure that the package that has been installed is the "
+            "one to 'Catch insensitive, inconsiderate writing'. This can be "
+            "verified by running `alex --help` and seeing what it does.")
+        try:
+            output = subprocess.check_output(("alex", "--help"),
+                                             stderr=subprocess.STDOUT)
+        except (OSError, subprocess.CalledProcessError):
+            return ("The `alex` package could not be verified. " +
+                    incorrect_pkg_msg)
+        else:
+            output = output.decode(sys.getfilesystemencoding())
+            if "Catch insensitive, inconsiderate writing" in output:
```

Comment:
@sils1297 would it work to check if 'which alex', resolving symlinks, yields an '.js' file?